apps/testing-javascript/src/data/sanity-import.ts

```diff
@@ -0,0 +1,559 @@
+// Running this script:
+//
+// Ensure you have `dataFilePath` set to the location of the formatted export
+// of Testing JavaScript data. Also, be sure to set `SANITY_EDITOR_TOKEN` in
+// your `.env.local` environment variables file.
+//
+// Execute the Mux Asset Upload script with:
+//
+// ```
+// npx ts-node --skipProject src/data/sanity-import.ts
+// ```
```

Comment:
This is the Sanity Import script that takes the testing javascript formatted JSON export and the Mux playback IDs and makes a series of API requests to generate all the Sanity objects